pkgs/development/python-modules/sphinx-fortran/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, future
+, numpy
+, sphinx
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-fortran";
+  version = "unstable-2022-03-02";
+
+  src = fetchFromGitHub {
+    owner = "VACUMM";
+    repo = pname;
+    rev = "394ae990b43ed43fcff8beb048632f5e99794264";
+    sha256 = "sha256-IVKu5u9gqs7/9EZrf4ZYd12K6J31u+/B8kk4+8yfohM=";
+  };
+
+
+  propagatedBuildInputs = [
+    future
+    numpy
+    sphinx
+    six
+  ];
+
+  pythonImportsCheck = [ "sphinxfortran" ];
+
+  # Tests are failing because reference files are not updated
+  # checkInputs = [
+  #   pytestCheckHook
+  # ];
```

```suggestion
  # Tests are failing because reference files are not updated
  doCheck = false;

  checkInputs = [
    pytestCheckHook
  ];
```
Could we rather only disable the failing test?

---

Wouldn't `doCheck = false` disable the Python import check?

All three tests are failing, if we disable all of the them, `pytest` will raise a warning which stops the build.

---

doCheck=false does not remove the name import check.
Why are the tests failing? should they? Requiring network access is a valid fail mode.

---

Tests are failing because the references files have not been updated after the last three commits. I created a PR to update them.